core/src/main/java/com/datastrato/gravitino/catalog/EntityCombinedSchema.java

```diff
@@ -23,6 +24,7 @@ public final class EntityCombinedSchema implements Schema {
 
   // Sets of properties that should be hidden from the user.
   private Set<String> hiddenProperties;
+  private boolean imported;
```

Comment:
I think you should add some comments about what it was used for.  What should we do if it's true or false?

Reply:
Done.

Reply:
Further information may be required, such as why we need to import it to the Gravitino-managed store.

Reply:
> Further information may be required, such as why we need to import it to the Gravitino-managed store.

@qqqttt123 do you take a look at these comments and check whether they need to be resolved?

Reply:
Let me update the doc.